STYLEGUIDE.md

```diff
@@ -390,9 +390,11 @@ On the frontend, some lint and style rules are enforced by [Stylelint](https://s
 
 Some key points:
 - Use SCSS modules over CSS-in-JS. The module file takes the name "my-component.module.scss" and lives in the same directory as the component.
-- Use kebab-case (not camelCase nor snake_case) for separating words in IDs, classes, mixins and filenames. 
-- Use `px` for small values (e.g. less than 4 px) and specific values (e.g. 87px). Use `em` for other values.
-  - To convert between `px` and `em` for our site, use a [Pixel to Em converter](https://www.w3schools.com/tags/ref_pxtoemconversion.asp) with 16 as the default pixel size.
+- Use kebab-case (not camelCase nor snake_case) for separating words in IDs, classes, mixins and filenames.
+- Use `px` for small values (e.g. less than 4 px) and specific values (e.g. 87px).
+- Use `rem` or `em` for other values like `font-size`, `margin`, and `padding`.
+  - Default to `rem`, but to quote [this article](https://www.digitalocean.com/community/tutorials/css-rem-vs-em-units) "use `em` units in places where the influence of nearby parent elements would make sense".
```

Comment:
Perhaps this is obvious to more design-expert people, but I don't know what this means. Would it be possible to provide an example? I think there are lots of us who don't have a great handle on styling patterns.

Reply:
Yeah! Since this already passed tests I'll merge this PR now and make a new PR w/ the margin-bottom on a heading element example.